update-web-playground.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 nix build .#web-playground
 rm -rf website/playground/wasm-app
-ln -s ../../result/post website/playground/wasm-app
+ln -s ../../result website/playground/wasm-app
```

Comment:
Is `website/playground/wasm-app` always a symlink? In which case, (FWIW) you could update it without deleting first with `ln -sf`.

Reply:
I actually tried that first, but I got a permission issue (at least on my Mac) when I tried to recreate it.